models/models.js

```diff
@@ -135,4 +135,8 @@ exports.deleteComment = (commentId) =>{
         return db.query(`DELETE FROM comments WHERE comment_id = $1;`, [commentId]).then((dbResponse)=>{return dbResponse})
     
     
+}
+
+exports.fetchUsers = ()=>{
+    return db.query(`SELECT * FROM users`).then((dbResponse)=>{return dbResponse.rows})
```

Comment:
You are destructuring in your test very well, I'd recommend consistency and use that here as well to destructure rows from the response.